cmd/bcpd/app/app.go

```diff
@@ -101,15 +102,15 @@ func Stack(issuer weave.Address) weave.Handler {
 // the given arguments. If you are not sure what to use
 // for the Handler, just use Stack().
 func Application(name string, h weave.Handler,
-	tx weave.TxDecoder, dbPath string, debug bool) (app.BaseApp, error) {
+	tx weave.TxDecoder, dbPath string, options *server.Options) (app.BaseApp, error) {
 
 	ctx := context.Background()
 	kv, err := CommitKVStore(dbPath)
 	if err != nil {
 		return app.BaseApp{}, err
 	}
 	store := app.NewStoreApp(name, kv, QueryRouter(), ctx)
-	base := app.NewBaseApp(store, tx, h, nil, debug)
+	base := app.NewBaseApp(store, tx, h, nil, options.Debug)
```

Comment:
Yes, I like this better.
The non-cmd packages get specific options, but the cmd dir gets them all to use at it sees fit